modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -77,13 +78,31 @@ For more information, see xref:search:quick-index-field-options.adoc#quick-field
 For example, you could add the text field that you used to generate your vector embeddings. 
 . Click btn:[Create Index].
 
+[#example]
+=== Example: Creating a Vector Search Index for Vector Search Query Examples 
+
+If you want to use the sample dataset for the examples in xref:run-vector-search-ui.adoc[] and xref:run-vector-search-sdk.adoc[], then you can xref:search:import-search-index.adoc[import] the following Search index definition into {page-ui-name}:
+
+[source,json]
+----
+include::example$sample-vector-search-index-payload.json[]
+----
+
+NOTE: Make sure you imported the sample dataset with the recommended settings. 
+
+This Vector Search index has a type mapping for a `color.rgb` collection and includes the following fields: 
+
+* The *brightness number* field, which is included in search results and supports sorting and faceting.
+* The *color string* field, which is included in search results, supports highlighting, phrase matching, and sorting and faceting.
+* The *description string* field, which is included in search results, supports highlighting, phrase matching, and sorting and faceting.
```

Comment:
Wondering if we could combine these two since they do the same thing?

```suggestion
* The *color string* and *description string* fields, which are included in search results and support highlighting, phrase matching, and sorting and faceting.
```